applications/RANSApplication/tests/cpp/test_incompressible_potential_flow_conditions.cpp

```diff
@@ -0,0 +1,282 @@
+{
+    const auto add_variables_function = [](ModelPart& rModelPart) {
+        rModelPart.AddNodalSolutionStepVariable(VELOCITY_POTENTIAL);
+        rModelPart.AddNodalSolutionStepVariable(VELOCITY);
+    };
+
+    using namespace RansApplicationTestUtilities;
+
+    auto& r_model_part = CreateScalarVariableTestModelPart(
+        rModel, "Element2D3N",
+        "RansIncompressiblePotentialFlowVelocityInlet2D2N",
+        add_variables_function, VELOCITY_POTENTIAL, 1);
+
+    // set nodal historical variables
+    RandomFillNodalHistoricalVariable(r_model_part, VELOCITY_POTENTIAL, -10.0, 10.0);
+    RandomFillNodalHistoricalVariable(r_model_part, VELOCITY, -5.0, 5.0);
+
+    RandomFillContainerVariable<ModelPart::ConditionsContainerType, array_1d<double, 3>>(
+        r_model_part, NORMAL, 1.0, 10.0);
+
+    return r_model_part;
+}
+
+ModelPart& RansIncompressiblePotentialFlowPressureBodyForce2D2N_SetUp(
+    Model& rModel)
+{
+    const auto add_variables_function = [](ModelPart& rModelPart) {
+        rModelPart.AddNodalSolutionStepVariable(VELOCITY_POTENTIAL);
+        rModelPart.AddNodalSolutionStepVariable(PRESSURE_POTENTIAL);
+        rModelPart.AddNodalSolutionStepVariable(DENSITY);
+        rModelPart.AddNodalSolutionStepVariable(BODY_FORCE);
+    };
+
+    using namespace RansApplicationTestUtilities;
+
+    ModelPart& r_model_part = CreateScalarVariableTestModelPart(
+        rModel, "Element2D3N",
+        "RansIncompressiblePotentialFlowPressureBodyForce2D2N",
+        add_variables_function, PRESSURE_POTENTIAL, 1, true, false);
+
+    // set nodal historical variables
+    RandomFillNodalHistoricalVariable(r_model_part, DENSITY, 2.0, 2.0);
+    RandomFillNodalHistoricalVariable(r_model_part, BODY_FORCE, -2.0, 2.0);
+    RandomFillNodalHistoricalVariable(r_model_part, VELOCITY_POTENTIAL, -10.0, 10.0);
+    RandomFillNodalHistoricalVariable(r_model_part, PRESSURE_POTENTIAL, -10.0, 10.0);
+
+    RandomFillContainerVariable<ModelPart::ConditionsContainerType, array_1d<double, 3>>(
+        r_model_part, NORMAL, 1.0, 10.0);
+
+    return r_model_part;
+}
+
+} // namespace
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2N_EquationIdVector,
+                          KratosRansFastSuite)
+{
+    // Setup:
+    Model model;
+    auto& r_model_part = RansIncompressiblePotentialFlowVelocityInlet2D2N_SetUp(model);
+
+    // Test:
+    RansApplicationTestUtilities::TestEquationIdVector<ModelPart::ConditionsContainerType>(r_model_part);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2N_GetDofList, KratosRansFastSuite)
+{
+    // Setup:
+    Model model;
+    auto& r_model_part = RansIncompressiblePotentialFlowVelocityInlet2D2N_SetUp(model);
+
+    // Test:
+    RansApplicationTestUtilities::TestGetDofList<ModelPart::ConditionsContainerType>(
+        r_model_part, VELOCITY_POTENTIAL);
+}
+
+KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2N_CalculateLocalSystem,
```

Comment:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(RansIncompressiblePotentialFlowVelocityInlet2D2NCalculateLocalSystem,
```

Reply:
This I would like to keep as it is because its a long name, having underscores gives little bit of readability :)